### PR TITLE
Dried fruits

### PR DESCRIPTION
**Pull request by Wayticus**
#### Summary
```SUMMARY: Content "Adds dried fruits items and recipes to make them."```

#### Purpose of change
To add raisins, and dried blueberries and such.

#### Describe the solution
Adds  dried fruits and recipes for making them.

#### Additional context: 
Added the fruits, based the vitC off of the existing levels but 1/3 of their current levels  due to loss in drying, and increased the kcal by 2.5 due to the concentration of the sugars etc. on most (some have more due to the added sugar in the pre bagged ones) for sake of consistency even though real world the vitC and kcal would vary by fruit.  
Added additional vitamins based on real world numbers, assuming roughly a 100g portion. Required 2 portions for consistency, of the base fruit for most cases due to weight loss when drying. 
The ones you can make have a shorter shelf life due to the lack of sulfur dioxide in the game, and not sure how the survivor would make it/what form it would be in, in order to add it to the fruit in a dehydrator.  Did not include the smoking rack in the recipes cause smokey fruit would taste odd and there are already the tasteless fruit flakes for that.
Gave most of them a stim of 1 due to the increased sugar content.

Hope I submitted this right.

### CODE REVIEW

**Comment by mlangsdorf** [comment]
Jenkins rebuild

**Comment by mlangsdorf** [comment]
Fixed your summary line.

**Comment by Wayticus** [comment]
Thank you for the fixes, what does the jenkins rebuild mean is it something I could have/should have done prior to submitting? 
Is there anything I have to  do/change because of the fail on Travis CI?  
This is all new to me so sorry for all the questions, and them multiple commits.

**Comment by Wayticus** [comment]
How do I edit this to be labeled a WIP, people asked about smoking rack drying, and seems there are recipes for smoked fruits out there, so despite personal taste, smokey fruit sounds icky,  I'm working on adding the smoking rack for this.

**Comment by Zireael07** [comment]
Edit button in top right of page, above the line count.

**Comment by Wayticus** [comment]
Thank you.

**Comment by Wayticus** [comment]
Seems the ability to put them in the smoking rack is hard coded though, and if I'm understanding the code right, possibly not, changing the base fruit items to be able to do so would effect the dry_fruit item since it calls on the sweet_fruit group.  I'm willing to try and learn/figure out the code to do so, since if this did go through it would kind of remove the need for the generic dry_fruit item.  Also willing to add the rest of the sweet_fruits group items to individual recipes for dried fruits.

File: comestibles.json [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+    "weight": 100,
+    "color": "yellow",
+    "spoils_in": "120 days",
+	"stim": 1,
+	"container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 345,
+    "description": "Dried apricots, good souce vitamin A",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried apricots, good source of vitamin A",
```

File: comestibles.json [hunk 2]
```diff
@@ -0,0 +1,586 @@
...
+	"name_plural": "dried apricots",
+    "weight": 100,
+    "color": "yellow",
+    "spoils_in": "60 days",
+	"stim": 1,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 345,
+    "description": "Dried apricots, good souce vitamin A",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried apricots, good source of vitamin A",
```

File: comestibles.json [hunk 3]
```diff
@@ -0,0 +1,586 @@
...
+	"name_plural": "dried blueberries",
+    "weight": 100,
+    "color": "blue",
+    "spoils_in": "60 days",
+	"stim": 1,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 345,
+    "description": "Dried blueberries, less antioxidants then fresh ones, but still good for you",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried blueberries, less antioxidants than fresh ones, but still good for you",
```

File: comestibles.json [hunk 4]
```diff
@@ -0,0 +1,586 @@
...
+    "weight": 100,
+    "color": "blue",
+    "spoils_in": "120 days",
+	"stim": 1,
+	"container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 345,
+    "description": "Dried blueberries, less antioxidants then fresh ones, but still good for you",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried blueberries, less antioxidants than fresh ones, but still good for you",
```

File: comestibles.json [hunk 5]
```diff
@@ -0,0 +1,586 @@
...
+    "weight": 100,
+    "color": "red",
+    "spoils_in": "120 days",
+    "stim": 1,
+	"container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 4,
+    "calories": 230,
+    "description": "Dried figs, odd looking but sweet ",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried figs, odd looking but sweet.",
```

File: comestibles.json [hunk 6]
```diff
@@ -0,0 +1,586 @@
...
+    "volume": 1,
+	"flags": [ "EDIBLE_FROZEN" ],
+    "charges": 3,
+	"fun": 3
+  },
+{
+    "type": "COMESTIBLE",
+    "id": "dried_dates",
+    "name": "dried dates",
+	"name_plural": "dreid dates",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
	"name_plural": "dried dates",
```

File: comestibles.json [hunk 7]
```diff
@@ -0,0 +1,586 @@
...
+    "name": "dried banana slices",
+	"name_plural": "dried banana slices",
+    "weight": 100,
+    "color": "red",
+    "spoils_in": "60 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 4,
+    "calories": 262,
+    "description": "Chewy dried banana slices, not to be confused with them artery clogging banana chips",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Chewy dried banana slices, not to be confused with them artery clogging banana chips.",
```

File: comestibles.json [hunk 8]
```diff
@@ -0,0 +1,586 @@
...
+	"name_plural": "dried banana slices",
+    "weight": 100,
+    "color": "red",
+    "spoils_in": "120 days",
+	"container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 4,
+    "calories": 262,
+    "description": "Chewy dried banana slices, not to be confused with them artery clogging banana chips",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Chewy dried banana slices, not to be confused with them artery clogging banana chips.",
```

File: comestibles.json [hunk 9]
```diff
@@ -0,0 +1,586 @@
...
+	"name_plural": "dried pineapple chunks",
+    "weight": 100,
+    "color": "red",
+    "spoils_in": "60 days",
+	"stim": 1,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 4,
+    "calories": 262,
+    "description": "Dried pineapple chunks",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried pineapple chunks.",
```

File: comestibles.json [hunk 10]
```diff
@@ -0,0 +1,586 @@
...
+    "weight": 100,
+    "color": "red",
+    "spoils_in": "120 days",
+	"stim": 2,
+	"container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 4,
+    "calories": 362,
+    "description": "Dried pineapple chunks, dusted in sugar, basicaly candy",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried pineapple chunks, dusted in sugar, basically candy.",
```

File: comestibles.json [hunk 11]
```diff
@@ -0,0 +1,586 @@
...
+    "weight": 100,
+    "color": "red",
+    "spoils_in": "120 days",
+	"stim": 1,
+	"container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 502,
+    "description": "Dried suagred mango slices, bright orange colour.",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried sugared mango slices, bright orange color.",
```

File: comestibles.json [hunk 12]
```diff
@@ -0,0 +1,586 @@
...
+	"name_plural": "dried kiwi slices",
+    "weight": 100,
+    "color": "brown",
+    "spoils_in": "120 days",
+	"stim": 1,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 2,
+    "calories": 297,
+    "description": "Dried kiwi sugared slices, almost fluorescent in colour.",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried kiwi sugared slices, almost fluorescent in color.",
```

File: comestibles.json [hunk 13]
```diff
@@ -0,0 +1,586 @@
...
+    "weight": 100,
+    "color": "yellow",
+    "spoils_in": "120 days",
+	"stim": 1,
+	"container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 2,
+    "calories": 455,
+    "description": "Dried papaya slices, vibrant orange in colour.",
```
Comment (Night-Pryanik) [inline comestibles.json]:
```suggestion
    "description": "Dried papaya slices, vibrant orange in color.",
```

File: item_groups.json [hunk 14]
```diff
@@ -0,0 +1,174 @@
+[
+{
+    "type" : "item_group",
+    "id" : "child_items",
+    "items":[
+	["b_dried_apple", 15]
+	]
```
Comment (Night-Pryanik) [inline item_groups.json]: Please use linter (http://dev.narc.ro/cataclysm/format.html) to properly format your changes.

File: modinfo.json [hunk 15]
```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "dried_fruits",
+    "name": "Dried Fruits",
+    "authors": [ "Wayticus" ],
+    "maintainers": [ "Wayticus" ],
+    "description": "Dried fruits, not to be confused with them tatseless dehydrated fruit flakes.",
```
Comment (Night-Pryanik) [inline modinfo.json]:
```suggestion
    "description": "Dried fruits, not to be confused with them tasteless dehydrated fruit flakes.",
```

**Comment by Night-Pryanik** [comment]
Why don't you add this as mainline content? I think it will fit nicely in vanilla.

**Comment by AMurkin** [comment]
Files are in the root directory.

**Comment by Treah** [comment]
May want to rename the commercially created or bagged type so they are listed different in the players inventory. As it stands I could have prunes I created and prunes I found but both are named the same but have different spoil timers.  Also personal preference for me would be to rename dried plums to prunes. They are pretty much called that everywhere, unless you were trying to make it easier for language conversions.  Also keep in mind that the default seasons are now 90 days long with 360 day years, when setting the correct spoil timers.

**Comment by Wayticus** [comment]
Thank you for some reason my test world was at 30 days instead of my default 91 day.

**Comment by Wayticus** [comment]
Actually after thinking a bit that I tried "bag of" for the pre-packged, but if you unload the bag it would  say "bag of", I just forgot to remove it on the raisins, and they should show as different items even after unloaded due to the rot time, I guess I can just forget the preservatives and make them all the same shelf life.

File: item_groups.json (hunk 14 above)
Comment (Wayticus) [reply item_groups.json]: Okay linted them up, didn't see that link earlier.

**Comment by Treah** [comment]
Yeah I really wish the game would combine things better with rot timers. I think it should combine things that are less then a day apart in difference and just round up. I am sure that may cause some exploited behavior but its at least worth a test and don't think it would be that bad.

**Comment by kevingranade** [comment]
Quick check through some sources indicates that a lot more than 66% of vitamins are lost in drying:
https://www.nytimes.com/2008/07/01/science/01qna.html
https://www.fooducate.com/app#!page=post&id=57A34CB6-0E84-FF41-4594-89C85040F63B

tl;dr drop vitamin content by 20x or more compared to fresh fruit.

**Comment by Wayticus** [comment]
The vitC levels were just based off the in game levels and I set it to 1/3 for consistency, I have since started PR for vitamin change for all fruits, I'll update the vitC levels on these dried ones to match what the sites I referenced listed for the dried variants.

**Comment by Wayticus** [comment]
VitC levels adjusted down, except the mango that went up those things are packed of vitC.
Based on the following sites

https://www.nutritionix.com,  https://nutritiondata.self.com/

**Comment by ZhilkinSerg** [comment]
Here how files should look like if branched from current master - [26748-dried_fruits.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/2703984/26748-dried_fruits.zip)

**Pull request closed without merging**